pkg/controllers/disruption/consolidation_test.go

```diff
@@ -236,6 +238,8 @@ var _ = Describe("Consolidation", func() {
 			// Execute command, thus deleting all nodes
 			ExpectReconcileSucceeded(ctx, queue, types.NamespacedName{})
 			Expect(len(ExpectNodeClaims(ctx, env.Client))).To(Equal(numNodes))
+			// Expect the cluster consolidated state to not change.
```

Comment:
sanity check: I'm assuming that this failed prior to this change?

Reply:
Should we test this in multiple places since we have 3 places where we are handling this logic but we only have a single test with emptiness where we are validating this behavior?

Reply:
The event is only emitted for consolidation when we have no allowed disruptions due to a blocking budget. I can add another test with multiple nodepools here